src/sentry/logging/formats.py

```diff
@@ -0,0 +1,15 @@
+"""
+sentry.logging.formats
+~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+"""
+This seems a little silly to be on it's own, but this could expand one day.
```

Comment:
I'd do this:

```python
class LoggingFormat(object):
    HUMAN = 'human'
    MACHINE = 'machine'
```

Then you can just access `LoggingFormat.HUMAN`. And move this into `logging/__init__.py`

Then you'd have an API like:

```python
from sentry.logging import LoggingFormat

LoggingFormat.HUMAN
```